test/methods.test.js

```diff
@@ -80,6 +91,17 @@ test('does not register /test/*/ when ignoreTrailingSlash is true', t => {
   )
 })
 
+test('defaults to built-in http METHODS for invalid option', t => {
+  t.plan(1)
+  const findMyWay = FindMyWay({ httpMethods: 'invalid' })
```

Comment:
I think this should throw :)

Reply:
i did it this way for backwards compat with people that might possibly (incorrectly) be passing an options object with httpMethods.  unlikely, but i opted to keep the current behavior which is nothing.  i agree it should throw though and i'm fine either way.

Reply:
Let's throw, we can't save everyone :)
In my opinion it's always better to be explicit and fail hard rather than silently fail or silently not doing what you were expecting.

Reply:
i'm happy to make the change and agree with the sentiment.  this is just changing the contract which always makes me nervous.  if you're not worried though, i'm not worried.